README.md

```diff
@@ -144,3 +144,25 @@ To learn React, check out the
 [last-commit-badge]: https://img.shields.io/github/last-commit/fnndsc/chris_ui.svg
 [repo-link]: https://github.com/FNNDSC/ChRIS_ui
 [code-size]: https://img.shields.io/github/languages/code-size/FNNDSC/ChRIS_ui
+
+
+### E2E TESTS ARE RAN USING CYPRESS
+
+## Prerequisites:
+- ChRIS_ultron_backend is running on `http://localhost:8000/api/v1/`
+- ChRIS_ui is running on `http://localhost:3000/`
+- You have Cypress installed using `npm install cypress`
```

Comment:
:bangbang:  `npm install cypress` is the wrong command, you simply need `npm install` which reads and installs the correct versoin from `package.json`